datasets/rotten_tomatoes/README.md

```diff
@@ -19,6 +19,55 @@ size_categories:
 - 1K<n<10K
 source_datasets:
 - original
+train-eval-index:
+- config: default
+  task: text-classification
+  task_id: binary_classification
+  splits:
+    train_split: train
+    eval_split: test
+  col_mapping:
+    text: text
+    label: target
+  metrics:
+    - type: accuracy
+      name: Accuracy
+    - type: f1
+      name: F1 macro
+      args:
+        average: macro
```

Comment:
For binary classification datasets, I'm not sure it makes much sense to compute macro / micro / weighted F1 scores. Instead we can just use binary and drop the rest of the entries below - WDYT?

```suggestion
        average: binary
```

Reply:
sure!